packages/plugin-button/src/createButtonPlugin.js

```diff
@@ -0,0 +1,54 @@
+import createToolbar from './toolbar';
+import { createBasePlugin, mergeStyles, decorateComponentWithProps } from 'wix-rich-content-common';
+
+import { BUTTON_TYPE } from './constants';
+
+import Styles from '../statics/styles/default-styles.scss';
+import ButtonComponent from './components/button-component';
+
+const createButtonPlugin = (config = {}) => {
+  const type = BUTTON_TYPE;
+  const {
+    helpers,
+    theme,
+    t,
+    anchorTarget,
+    relValue,
+    isMobile,
+    [type]: settings = {},
+    ...rest
+  } = config;
+  const styles = mergeStyles({ styles: Styles, theme });
+  const buttonProps = {
+    className: theme,
+    anchorTarget,
+    relValue,
+    settings,
+  };
+
+  const buttonEntityComponent = decorateComponentWithProps(ButtonComponent, buttonProps);
+
+  return createBasePlugin({
+    component: buttonEntityComponent,
```

Comment:
In `develop` branch we're passing `theme`, `anchorTarget`, `relValue` and `settings` through `context` so this is unnecessary.  remove it and we will make the necessary adjustments before merging.
```suggestion
    component: ButtonComponent,
```